client/components/data/query-jetpack-user-connection/index.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingJetpackUserConnection } from 'state/selectors';
+import { requestJetpackUserConnectionData } from 'state/jetpack/connection/actions';
+
+class QueryJetpackUserConnection extends Component {
+	static propTypes = {
+		siteId: PropTypes.number,
+		// Connected props
+		isRequesting: PropTypes.bool,
+		requestJetpackUserConnectionData: PropTypes.func.isRequired,
+	}
+
+	componentDidMount() {
+		this.request( this.props );
```

Comment:
Yes, but if the id is `0` the request won't ever be made. This is probably the id we would be using when trying Calypso in wp-admin for a single site installation.